xblock/core.py

```diff
@@ -746,6 +748,11 @@ def parse_xml(cls, node, runtime, keys):
             keys (:class:`.ScopeIds`): The keys identifying where this block
                 will store its data.
         """
+        if is_pointer_tag(node):
+            # new style:
```

Comment:
I think we can remove the comment and add better docs on load_definition_xml method